docset/winserver2025-ps/smbshare/Get-SmbShare.md

```diff
@@ -304,11 +335,13 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -LeasingMode
-Specifies SMB leasing and oplock behaviors.
+### -IsolatedTransport
```

Comment:
I need to look at this code a bit more for the help, standby on this one.

Reply:
@nedpyle - as discussed

Reply:
Help:

"Treat this share as if it is a separate file server instance. Clients will create a new set of connections to that share." 

or to this effect. This was built for Azure but we can't hide it, there's no good reason for customers to use it and we don't recommend they use it.